pkg/cmd/subscription/get/get.go

```diff
@@ -0,0 +1,91 @@
+package get
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type GetOptions struct {
+	HttpClient func() (*http.Client, error)
+	BaseRepo   func() (ghrepo.Interface, error)
+	IO         *iostreams.IOStreams
+
+	Repository string
+}
+
+func NewCmdGet(f *cmdutil.Factory, runF func(*GetOptions) error) *cobra.Command {
+	opts := &GetOptions{
+		IO:         f.IOStreams,
+		HttpClient: f.HttpClient,
+		BaseRepo:   f.BaseRepo,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Get subscription information to the repository",
+		Long: heredoc.Doc(`
+			Get subscription information to the repository.
+
+			Without any arguments, this command prints out the subscription information
+			for the current repository.
+
+			The possible subscription states are:
+			- all: you are watching the repository and notified of all notifications.
+			- ignore: you are watching the repository but not notified of any notifications.
+			- unwatched: you are not watching the repository or custom notification is set.
+		`),
```

Comment:
🤔 how does this align with other documentation like [`gh repo view`](https://cli.github.com/manual/gh_repo_view), [`gh issue view`](https://cli.github.com/manual/gh_issue_view), etc?

```suggestion
		Short: "Display whether subscribed to a GitHub repository for notifications",
		Long: heredoc.Docf(`
			Display whether subscribed to a GitHub repository for notifications.

			With no argument, the subscription information for the current repository is displayed.

			The possible subscription states are:
			- %[1]sall%[1]s: you are watching the repository and notified of all notifications.
			- %[1]signore%[1]s: you are watching the repository but not notified of any notifications.
			- %[1]sunwatched%[1]s: you are not watching the repository or custom notification is set.
		`, "`"),
```

- emphasizes subscription states
- aligns with other commands' documentation